docs/self-managed/deployment/helm/configure/authentication-and-authorization/external-idp-via-internal-keycloak.md

```diff
@@ -0,0 +1,163 @@
+---
+id: external-idp-via-internal-keycloak
+sidebar_label: External IdP via Internal Keycloak
+title: Set up the Helm chart with an external IdP through the internal Keycloak
+description: Learn how to configure an external identity provider to authenticate users while using the internal Keycloak as an identity broker.
+---
+
+This guide shows you how to configure the internal Keycloak instance to act as an identity broker, delegating authentication to an external identity provider (IdP) such as a corporate OIDC provider, SAML, LDAP, or Active Directory.
+
+This setup allows you to:
+
+- Use your organization's existing identity provider for user authentication
+- Retain the internal Keycloak for Camunda's OIDC integration
+- Manage user authorization through Camunda's identity systems
+
+## Prerequisites
+
+- A Camunda 8 deployment with internal Keycloak enabled. See [Set up the Helm chart with the internal Keycloak instance](/self-managed/deployment/helm/configure/authentication-and-authorization/internal-keycloak.md).
+- Access to your external IdP's configuration (client credentials, endpoints, etc.)
+
+## Configure the external identity provider
+
+Complete the following steps to integrate your external IdP:
+
+1. [Add your external IdP to Keycloak](#add-your-external-idp-to-keycloak)
+1. [Configure identity provider mappers](#configure-identity-provider-mappers)
+1. [Configure Orchestration Cluster Identity](#configure-orchestration-cluster-identity)
+1. [Configure Management Identity access](#configure-management-identity-access-optional) (optional)
+
+### Add your external IdP to Keycloak
+
+Configure Keycloak to use your external identity provider by following the [Configure an external IdP using Keycloak](/self-managed/components/management-identity/configuration/configure-external-identity-provider.md) guide.
+
+### Configure identity provider mappers
```

Comment:
For the mappers that we are creating here, it would help me as a reader if we give a brief explanation (one sentence) what their effect is (e.g. the group mapper adding the group to the access tokens of all external users). Being a relative Keycloak newbie, I was able to follow the guide and make it work but felt I don't quite understand what I had configured and how it works.